### ISSUE TITLE
va-file-input: read only additional inputs filetype is not perceivable by screen readers

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/vets-design-system-documentation#4721]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

The "What kind of file is this?" and doc type text are skipped / not readable in browse mode for screen readers

## What I expected to happen

The screen reader should be able to land on the text and hear what kind of file type the uploaded file is.

## Reproducing

- CSS Library version: Current
- Device: (e.g. iPhone 8, Macbook) VoiceOver & Safari on desktop and iOS / JAWS + Chrome / NVDA + Firefox
- Browser: (e.g. Firefox, IE 11)

Steps to reproduce:

1. Go to https://design.va.gov/storybook/?path=/story/uswds-va-file-input--read-only-with-additional-inputs
2. Use one of the screen reader / browser combinations listed above
3. Navigate the component in browse mode
4. You will hear the filename and file size
5. You will not be able to land on "What kind of file is this?" and the file type text. 

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other

**Comment by jeana-adhoc** [comment]
This is still happening. We are setting `inert` on the component which makes it *not* discoverable by screen readers.

https://a11ysupport.io/tech/html/inert_attribute
* For keyboard: inert content will be removed from the tab order and it is not possible to trigger any functionality.
* For screen readers: inert content will be removed from the tab order and it is not possible to trigger any functionality.
* For voice control: it is not possible to trigger any inert functionality.
* It is not possible to read anything contained within an element that has the inert attribute.

We need to figure out a different way to display a "read only" state.
